### DISCUSSION TITLE
make check failed

### DISCUSSION THREAD

**Discussion by sharemovie121** [jedisct1/libsodium#1286] [Q&A]
After running
git clone https://github.com/jedisct1/libsodium.git && cd libsodium && ./autogen.sh -s && ./configure && make check, I got the following errors:

ERROR: aead_aegis128l
ERROR: aead_aegis256

Any idea  how to fix it?

**Comment by jedisct1** [comment]
Don't use development code, that explicitly prints:

```
*** This is unstable, untested, development code.
It might not compile. It might not work as expected.
It might be totally insecure.
Do not use this except if you are planning to contribute code.
Use releases available at https://download.libsodium.org/libsodium/releases/ instead.
Alternatively, use the "stable" branch in the git repository.
```

before compiling every single file?
